Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: try-runtime-turing
```

Comment:
Seems unlikely we'd ever have a file named `try-runtime-turing`, but probably good practice.

Also not sold on adding a `Makefile`.